addons/web_editor/static/src/js/editor/odoo-editor/src/commands/deleteBackward.js

```diff
@@ -125,6 +126,29 @@ HTMLElement.prototype.oDeleteBackward = function (offset, alreadyMoved = false,
             return;
         }
 
+        /** If we are at the beninning of a block node,
+         *  And the previous node is empty, remove it.
+         *
+         *   E.g. (previousEl == empty)
+         *        <p><br></p><h1>[]def</h1> + BACKSPACE
+         *   <=>  <h1>[]def</h1>
+         *
+         *   E.g. (previousEl != empty)
+         *        <h3>abc</h3><h1>[]def</h1> + BACKSPACE
+         *   <=>  <h3>abc[]def</h3>
+        */
+        const previousElementSiblingClosestBlock = closestBlock(this.previousElementSibling);
+        if (
+            previousElementSiblingClosestBlock &&
+            (isEmptyBlock(previousElementSiblingClosestBlock) ||
+                previousElementSiblingClosestBlock.textContent === '\u200B') &&
+            !['TR','TD','TABLE','TBODY','UL','OL','LI'].includes(this.nodeName)
```

Comment:
I'm not sure we want yet another list of tags here... Why this list in particular ? Don't we have a similar list already ? Couldn't it be some kind of constant ? Maybe not, maybe the list is very specific to the particular use case at hand here and it makes sense, but I'm doutful of it so I'd rather ask.

Reply:
I'll create a contstant.